http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/sam/SamAuthProvider.scala

```diff
@@ -152,6 +152,21 @@ class SamAuthProvider[F[_]: Effect: Logger](samDao: SamDAO[F],
     checkPersistentDiskPermissionWithProjectFallback(samResource, authorization, action, googleProject)
   }
 
+  def getRuntimeActions(samResource: SamResource, userInfo: UserInfo)(
```

Comment:
Since this method is Runtime-specific, should it take `RuntimeSamResource` instead?